src/Packetery/Module/Order/Controller.php

```diff
@@ -56,36 +55,26 @@ class Controller extends WP_REST_Controller {
 	 */
 	private $orderRepository;
 
-	/**
-	 * Calculator.
-	 *
-	 * @var Calculator
-	 */
-	private $calculator;
-
 	/**
 	 * Controller constructor.
 	 *
 	 * @param Modal            $orderModal       Modal.
 	 * @param ControllerRouter $controllerRouter Router.
 	 * @param PacketSubmitter  $packetSubmitter  Packet submitter.
 	 * @param Order\Repository $orderRepository  Order repository.
-	 * @param Calculator       $calculator       Calculator.
 	 */
 	public function __construct(
 		Modal $orderModal,
 		ControllerRouter $controllerRouter,
 		PacketSubmitter $packetSubmitter,
-		Order\Repository $orderRepository,
-		Calculator $calculator
+		Order\Repository $orderRepository
 	) {
 		$this->orderModal      = $orderModal;
 		$this->router          = $controllerRouter;
 		$this->namespace       = $controllerRouter->getNamespace();
 		$this->rest_base       = $controllerRouter->getRestBase();
 		$this->packetSubmitter = $packetSubmitter;
 		$this->orderRepository = $orderRepository;
-		$this->calculator      = $calculator;
```

Comment:
Ta kalkulace se prováděla proto aby uživatel měl možnost automatizovaně přepočítat ručně nastavenou hodnotu tím že ji v modálu smaže. Teď je tam logika která smazanou hodnotu nastaví zpět. Proč se to měnilo? Prosím konzultovat s Jaroslavem a případně opravit.

Reply:
To je chyba, opraveno.